src/components/VmDetails/cards/SnapshotsCard/SnapshotItem.js

```diff
@@ -121,9 +122,10 @@ class SnapshotItem extends React.Component {
           key='info'
         >
           <a id={`${this.props.id}-info`}>
-            <OverlayTooltip placement={this.state.isMobile ? 'right' : 'left'} id={`${this.props.id}-info-tt`} tooltip={msg.details()}>
-              <Icon type='pf' name='info' />
-            </OverlayTooltip>
+            <InfoToolTip
```

Comment:
Ok, with all the placements removed, does the component auto-align based on available space?  Have you tested it with a narrow/phone screen?

Reply:
no the component isn't auto align based on available space, like PF3 component:
![image](https://user-images.githubusercontent.com/64131213/97982553-a7ca5d80-1ddc-11eb-88ab-77ed85968853.png)
do we want to edit the PF3 components?

Reply:
Narrow screens are more like best effort.  The Snapshot item list tooltips had handling for narrow screens that was removed.  So, if specifically those tooltips look ok with that code removed on a narrow screen, it's ok.  We just don't want to loose functionality.

Reply:
It looks OK:
![image](https://user-images.githubusercontent.com/64131213/98009924-c6d9e700-1dfe-11eb-8ceb-0af607e0ae4d.png)